### ISSUE TITLE
ember/require-return-from-computed false positive

### ISSUE THREAD

**Issue by efx** [ember-cli/eslint-plugin-ember#654]
Somewhat related to #492 
The following style triggers the warning:
```js

	someProperty: computed({
		get() {
			return 'hi';
		},
		set() {
			this.setProperties({
				isBad: false,
				isOk: false
			});
		}
	}),
```
It is confusing that the `set` would require returning anything. Should the rule be updated to ignore that case? Or what is the rationale for requiring this rule?

**Comment by rwjblue** [comment]
This is exactly the case the rule is trying to catch. With the `Ember.computed` API, the value you return from your setter is _exactly_ the value that will be returned for the next `get`. In other words, it _will not_ call your getter and it will instead use whatever your setter returned.

**Comment by efx** [comment]
oh, that makes a lot more sense. Thank you for explaining @rwjblue !
